lib/src/authorization_code_grant.dart

```diff
@@ -112,7 +116,8 @@ class AuthorizationCodeGrant {
           {this.secret,
           String delimiter,
           bool basicAuth: true,
-          http.Client httpClient})
+          http.Client httpClient,
+          Map<String, dynamic> this.getParameters(String contentType, String body)})
```

Comment:
Document this in the constructor docs. Same everywhere else we take it as a parameter.